modules/installation-network-user-infra.adoc

```diff
@@ -266,31 +296,91 @@ A working configuration for the Ingress router is required for an
+This section provides an example API and application ingress load balancer configuration that meets the load balancing requirements for user-provisioned clusters. The sample is an `/etc/haproxy/haproxy.cfg` configuration for an HAProxy load balancer. The example is not meant to provide advice for choosing one load balancing solution over another.
 
-* `00:05:69:00:00:00` to `00:05:69:FF:FF:FF`
-* `00:0c:29:00:00:00` to `00:0c:29:FF:FF:FF`
-* `00:1c:14:00:00:00` to `00:1c:14:FF:FF:FF`
-* `00:50:56:00:00:00` to `00:50:56:FF:FF:FF`
-
-If a MAC address outside the VMware OUI is used, the cluster installation will
-not succeed.
-endif::vsphere[]
+[NOTE]
+====
+In the example, the same load balancer is used for the Kubernetes API and application ingress traffic. In production scenarios you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
+====
 
-ifdef::vsphere[]
-:!vsphere:
-endif::[]
+.Sample API and application ingress load balancer configuration
+[%collapsible]
+====
+[source,text]
+global
+  log         127.0.0.1 local2
+  pidfile     /var/run/haproxy.pid
+  maxconn     4000
+  daemon
+defaults
+  mode                    http
+  log                     global
+  option                  dontlognull
+  option http-server-close
+  option                  redispatch
+  retries                 3
+  timeout http-request    10s
+  timeout queue           1m
+  timeout connect         10s
+  timeout client          1m
+  timeout server          1m
+  timeout http-keep-alive 10s
+  timeout check           10s
+  maxconn                 3000
+frontend stats
+  bind *:1936
+  mode            http
+  log             global
+  maxconn 10
+  stats enable
+  stats hide-version
+  stats refresh 30s
+  stats show-node
+  stats show-desc Stats for ocp4 cluster <1>
+  stats auth admin:ocp4
+  stats uri /stats
+listen api-server-6443 <2>
+  bind *:6443
+  mode tcp
+  server bootstrap bootstrap.ocp4.example.com:6443 check inter 1s backup <3>
+  server master0 master0.ocp4.example.com:6443 check inter 1s
+  server master1 master1.ocp4.example.com:6443 check inter 1s
+  server master2 master2.ocp4.example.com:6443 check inter 1s
+listen machine-config-server-22623 <4>
+  bind *:22623
+  mode tcp
+  server bootstrap bootstrap.ocp4.example.com:22623 check inter 1s backup <3>
+  server master0 master0.ocp4.example.com:22623 check inter 1s
+  server master1 master1.ocp4.example.com:22623 check inter 1s
+  server master2 master2.ocp4.example.com:22623 check inter 1s
+listen ingress-router-443 <5>
+  bind *:443
+  mode tcp
+  balance source
+  server worker0 worker0.ocp4.example.com:443 check inter 1s
+  server worker1 worker1.ocp4.example.com:443 check inter 1s
+listen ingress-router-80 <6>
+  bind *:80
+  mode tcp
+  balance source
+  server worker0 worker0.ocp4.example.com:80 check inter 1s
+  server worker1 worker1.ocp4.example.com:80 check inter 1s
+====
```

Comment:
(Collapsible content suggestions) Replace `====` here with `----` to close the code block.

Reply:
Thank you!